packages/react-color-picker/src/components/ColorChannelSlider.tsx

```diff
@@ -84,18 +88,14 @@ function ColorChannelBG({ channel, color, displayMin, displayMax, min, max }: Co
   }
 
   const range = (displayMax ?? max) - (displayMin ?? min);
-  let leftColor = { ...color.original, [channel]: min, alpha: 1 } as WebGLColor;
-  if (!RGB_COLORSPACES.includes(color.original.mode)) {
-    leftColor = COLORSPACES.rec2020.converter(leftColor);
-  }
-  let rightColor = { ...color.original, [channel]: max, alpha: 1 } as WebGLColor;
-  if (!RGB_COLORSPACES.includes(color.original.mode)) {
-    rightColor = COLORSPACES.rec2020.converter(rightColor);
-  }
+  const leftColor = { ...color.original, [channel]: displayMin ?? min };
+  const rightColor = { ...color.original, [channel]: displayMax ?? max };
+  const leftOklab = useMemo(() => withAlpha(toOklab(leftColor) as Oklab) as Oklab, [color.css]);
```

Comment:
The `useColor()` hook I made because I’M sO sMArT could be better, because adjustments like this are hard (specifically, this nonsense is because the original color may be in another space, so we want to adjust the color _before_ converting, which isn’t currently possible.

I may just ditch it in favor of raw Culori methods, but for now this is a bandaid to not refactor that.

I’m honestly really happy with the performance of `useColor` as it’s designed to be used. My problem is I need more from its design 😓